docs/src/administration/users.md

```diff
@@ -1,65 +1,53 @@
+|Project Admin | Users who can configure project settings, administer environment permissions, push code, and execute actions on all project environments.|                                                                                                                                                                                      
 
-Grant user permissions to the entire project:
-* **Project Viewer** - A project viewer can view all environments of the project.
-* **Project Administrator** - A project administrator can change settings, push code and execute actions on all environments of the project.
+### Permissions per environment type
 
-You can also define user access per environment type (Production, Staging, Development). Each permission level progressively increases access to the environments part of a given environment type:
+Environment types offer a way to group environments (production, staging, and development) together. If you group your environments, you can grant user permissions per environment type at the project level.
 
-* **Viewer** - Viewer can view all environments of this type.
-* **Contributor** - Contributor can push code and branch all environments of this type.
-* **Administrator** - Administrator can change settings and execute actions on all environments of this type.
+A few things to consider:
 
-{{< note theme="warning" title="Important" >}}
-After you add or remove a user from a project or an environment type, you will need to trigger a redeploy to propagate the access changes to each environment.
-You can redeploy by using the CLI command `platform redeploy` or the `Redeploy` button in the management console. 
-{{< /note >}}
-
-If you want your contributors to be able to see everything, but only commit to environments of a certain type, set their permission as **Project Viewer** for the whole project and update their permission on that environment type to **Contributor**.
-
-{{< note >}}
-The project owner - the person licensed to use Platform.sh - has the project administrator role. The project owner is the only user that can make changes to the project plan. 
-{{< /note >}}
-
-## SSH access
-
-By default, everyone with access equal or greater than **Contributor** can access the project through SSH. 
-
-To customize who can use SSH, [set the `access` key](/configuration/app/access.md) in your `.platform.app.yaml` file.
-
-## Manage user permissions in the console
-
-From your list of projects, select the project where you want to view or edit user permissions. Click the Settings tab at the top of the page, then click the `Access` tab on the left to see the users in the project and their roles.
-
-![Project user management screenshot](/images/management-console/settings-project-access.png)
+* Environments can share the same type.
```

Comment:
Does this mean there can be more than one environment in a type? Isn't that implied by the idea of "grouping" them? Or is there something more here that I'm missing? Wouldn't be the first time :grin:

Reply:
What I understand by this is that you can have multiple staging or developement environments so a type is "staging type"or "developement type"

Reply:
There can be multiple environments that are of the same type. My question was about what the phrasing here was supposed to add. I feel like "share" with "type" feels slightly confusing, or at least it's not what I'm used to reading with types. It seems to be included in the idea of types being a group.